setup.py

```diff
@@ -48,7 +48,7 @@ def get_packages(package):
     packages=get_packages("databases"),
     package_data={"databases": ["py.typed"]},
     data_files=[("", ["LICENSE.md"])],
-    install_requires=['sqlalchemy<1.14', 'aiocontextvars;python_version<"3.7"'],
+    install_requires=['sqlalchemy<1.4', 'aiocontextvars;python_version<"3.7"'],
```

Comment:
@tomchristie sorry to bother. I'm running into problems because of this. Any insight into when sqlalchemy>1.3 will be supported? What can I possibly do to help?

Reply:
Not sure - I've not *personally* got the extra time for this at the moment.
There's some work on supporting 1.4 in #299 - You might want to start by trying that out locally and confirming if it works for you, and if it does then helping review any remaining points that's blocking that pull request from being accepted.

Reply:
thanks!